migrations/create_webmaker_db.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE IF NOT EXISTS "users"
+(
+  id serial NOT NULL,
+  username varchar(20) NOT NULL,
```

Comment:
You can drop the length here and just enforce in Joi. Generally best to let pg do what's best (unlike mysql) with `varchar`s.